app/main/views.py

```diff
@@ -1,80 +1,87 @@
-import requests
 import os
+import requests
 import re
+from flask import render_template, request, redirect, url_for
 from . import main
-from flask import json, render_template, Response, request, redirect
+from .helpers.validation_tools import Validate
+from .helpers.content import Content_loader
+from .helpers.service import Service_loader
+
+
+service = Service_loader(
+    os.getenv('DM_API_URL'),
+    os.getenv('DM_API_BEARER')
```

Comment:
Based on a conversation with @quis and @TheDoubleK we've decided to use a common convention for ENV variable names: `DM_<app name>_<variable name>`.

So we'll rename `DM_API_BEARER` to `DM_ADMIN_FRONTEND_API_AUTH_TOKEN` so that we can use a different API token for different apps. We can omit the `<app name>` part for variables that will be used by more than one app, like `DM_API_URL`.